apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/page.tsx

```diff
@@ -14,16 +14,13 @@ import { ExplorersSection } from "./components/server/explorer-section";
 
```

Comment:
TLDR; is - In the Chain landing page, we don't really care about filtering the popular contracts - so I removed so we can statically render the chain page, 

Added force-dynamic for /popular and /trending pages where we allow filtering by search params